MicroProfile-Fault-Tolerance/tck-runner/pom.xml

```diff
@@ -96,19 +37,37 @@
             <version>${microprofile.fault-tolerance.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>${testng.version}</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
-    <profiles>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>${basedir}/src/test/resources/no-ext-libs-tck-suite.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
 
+    <profiles>
         <profile>
             <id>stable</id>
-
+            <activation>
```

Comment:
This is only for Jenkins' benefit. This should not be activated by default.

Reply:
Similar to discussion on https://github.com/payara/patched-src-microprofile1.2-samples/pull/7#discussion_r243356674. I'm in favour of defaulting to stable profiles. What are your thoughts?

Reply:
I agree projects should be stable by default. Though running a project with -P \!stable isn't very intuitive. My preference is for a "also-run-unstable-tests"